### PR TITLE
Blob Archiver API

### PR DESCRIPTION
**Pull request by syntrust**
See [the spec](https://github.com/ethstorage/es-node/blob/main/ethstorage/archiver/spec.md) for details.

### CODE REVIEW

File: ethstorage/archiver/blob.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		"}, "
+}
+
+type alias BlobSidecar
+type blobSidecarAlias struct {
+	Index         string `json:"index"`
+	KZGCommitment string `json:"kzg_commitment"`
+	KZGProof      string `json:"kzg_proof"`
+	Blob          string `json:"blob"`
+	*alias
```
Comment (qzhodl) [inline ethstorage/archiver/blob.go:42]: Why we want to introduce this field?

Comment (syntrust) [reply ethstorage/archiver/blob.go]: removed for simplicity

**Review by qzhodl — APPROVED** [review]
lgtm

File: ethstorage/archiver/api.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+		return nil, hErr
+	}
+	a.logger.Info("BeaconID to execution block number", "beaconID", id, "elBlock", elBlock)
+
+	blobsInBeacon := len(kzgCommitsAll)
+	if indices != nil {
+		for _, index := range indices {
+			if int(index) >= blobsInBeacon {
+				// beacon API will ignore invalid indices and return all blobs
+				indices = nil
```
[the remark this answers is not in the document]
Comment (syntrust) [reply ethstorage/archiver/api.go:57]: As the comment says we do this to be aligned with the beacon API. Although `newIndicesError` could be an option 
 we may consider based on later tests with op stack testnet, this is an edge case anyway.

**Review by ping-ke — APPROVED** [review]
